src/components/atoms/Page/Component.tsx

```diff
@@ -10,68 +10,72 @@ import {
   ScrollView,
   TouchableOpacity,
   KeyboardAvoidingView,
+  KeyboardAvoidingViewProps,
+  ViewProps,
 } from 'react-native';
 
-export type PageProps = View &
-  KeyboardAvoidingView & {
-    scrollEnabled?: boolean;
-    children?: React.ReactNode;
-    SafeAreaView: any;
-    style?: StyleProp<ViewStyle>;
-    backgroundColor?: string;
-    statusBar?: 'light-content' | 'dark-content';
-    unsafe?: boolean;
-    hidden?: boolean;
-    statusColor?: string;
-    safeArea?: boolean;
-    /**
-     * Enable to draw behind status bar android
-     * @default false
-     */
-    draw?: boolean;
+import type { SafeAreaView } from 'react-native-safe-area-context'
 
-    /**
-     * Enable to custom background color of inset bottom on Iphone
-     * @default false
-     */
-    insetTop?: boolean;
+export interface PageProps extends ViewProps, KeyboardAvoidingViewProps {
+  scrollEnabled?: boolean;
+  children?: React.ReactNode;
+  SafeAreaView: typeof SafeAreaView;
+  style?: StyleProp<ViewStyle>;
+  backgroundColor?: string;
+  statusBar?: 'light-content' | 'dark-content';
+  unsafe?: boolean;
+  hidden?: boolean;
+  statusColor?: string;
+  safeArea?: boolean;
+  /**
+   * Enable to draw behind status bar android
+   * @default false
+   */
+  draw?: boolean;
 
-    /**
-     * Enable to custom background color of inset bottom on Iphone
-     * @default false
-     */
-    insetBottom?: boolean;
```

Comment:
do we need to update in our `jitera-frontend`? 🤔

Reply:
Yes, I'll update as well